txpipe/twopoint_fourier.py

```diff
@@ -221,11 +223,85 @@ def load_maps(self):
         syst_nc = None
         syst_wl = None
 
+        
+        # Load HEALPix systematics maps
+        syst_nc = self.config['syst_nc']
+        syst_wl = self.config['syst_wl']
+        if syst_nc:  
+            print('Deprojecting systematics maps for number counts')
+            n_systmaps = 0
+            s_maps = []
+            systmaps_dir = self.config['systmaps_nc_dir']
+            systmaps_path = pathlib.Path(systmaps_dir)
+            try:
```

Comment:
should this try statement be inside the loop so that if one map doesn't work the others can still deproject?

Reply:
This was here to catch the problem of no map files in the given directory. If one of the maps is a .fits file that is not a proper healpix maps file `healpy.read_map` will give an error and the job will stop. So I am not sure about moving it.